modules/tinymce/src/core/main/ts/api/ui/Registry.ts

```diff
@@ -106,6 +106,9 @@ const registry = () => {
      * @param {String} name Unique name identifying the new icon.
      * @param {svgData} string The SVG data string the browser will use to render the SVG icon.
      * @return {void} void
+     * @example
+     * //To add a simple triangle icon:
+     * editor.ui.registry.addIcon('triangleUp', '<svg height="10" width="10"><path d="M5 0 L10 10 L0 10 Z" /></svg>' );
```

Comment:
It might be better to use `24` as the width/height here, as that's what we recommend for all icons IIRC (@lostkeys would know for sure).

Reply:
Yes please use `24` as that is the default icon size.

Reply:
Resized to 24 x 24 in https://github.com/tinymce/tinymce/pull/5835/commits/e9a85e08fb5427c187e41131c24eb3c9444d3b44